introduction.md

```diff
@@ -7,4 +7,8 @@ React.render(
   document.getElementById('todoapp')
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?